Gujarati/મોસમ/વરસાદ

```diff
@@ -0,0 +1 @@
+વરસાદ નો સમય,ગરમ ગરમ ભજીયા અને પરિવાર ની મૌજ મસ્તી, એક માણસ ને વધારે સુ જુએ.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.